packages/catalog-search/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@edx/frontend-enterprise-catalog-search",
-  "version": "0.1.4",
+  "version": "0.1.0",
```

Comment:
Did you want to go back to 0.1.0?

Reply:
Ooooh, so I had mistakenly thought these were bumped from running `npm pack` without the `--dry-run` option. But, actually there were actually automated releases due to package upgrades by Renovate. I've disabled Renovate's automerge on patch releases for the time being.

Great catch!